python/pip.bzl

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
```

Comment:
Would probably be nice to add documentation of this functionality to a `README.md`?

Reply:
Docs updated.